modules/multi-architecture-scheduling-examples.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: CONCEPT
+[id="multi-architecture-scheduling-examples_{context}"]
+
+= Sample multi-architecture node workload deployments 
+
+There are various cases to consider when you are scheduling workloads on a cluster with compute nodes of different architectures. 
+
+Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on only a set of nodes with architectures supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod template specification.
```

Comment:
```suggestion
Using node affinity to schedule workloads on a node:: To allow a workload to be scheduled on only a set of nodes with architectures supported by its images, you can set the `spec.affinity.nodeAffinity` field in your pod's template specification.
```

Reply:
"Allow" might be a form of anthropomorphism if it is directed at the user. But maybe in this case, the verb is directed at an object-to-object relationship? 

See: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

Reply:
"its" might get lost among the nouns (workload, nodes, architectures), maybe be more specific (that are supported byt the workload's images)?

Reply:
Ah gotcha, yeah I see what you mean, lemme re word that. Thank you!